views/guide_colour.html

```diff
@@ -379,6 +379,10 @@ <h4 class="heading-small">GOV.UK blue</h4>
 
   <h3 class="heading-medium" id="examples">Examples</h3>
   <h4 class="heading-small">Transaction end page</h4>
+  
+  <p>
+    Note that the white text on turquoise background only passes accessibility standards if the font size is at least 18px, or 14px bold. 
+  </p>
```

Comment:
'accessibility standards' is a little vague, could this be revised to "only meets [
[WCAG2 Level AA](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast), with a contrast ratio of 3:1 for large text - when the font size is at least 18px, or 14px bold.